src/Components/NavBar/Menus/NewMobileNavMenu.tsx

```diff
@@ -173,12 +173,12 @@ const AuthenticateLinks = () => {
     <Box>
       <Separator my={1} color={color("black10")} />
       <MobileLink
-        href={`/sign_up?intent=signup&signupIntent=signup&trigger=click&contextModule=Header`}
+        href={"/sign_up?intent=signup&trigger=click&contextModule=Header"}
       >
```

Comment:
This also removes the  `signupIntent` param because we [no longer need it](https://github.com/artsy/reaction/blob/2855bc6d270138eb6b2848253e3783ed711ab12e/src/Components/NavBar/Menus/MobileNavMenu.tsx#L69) as a result of @eessex's [work](https://github.com/artsy/reaction/pull/3264) to type `openAuthModal`.